### PR TITLE
Fix readme images size

### PR DESCRIPTION
**Pull request by evidolob**
PR just set proper images size, to have proper aspect ratio.
Fix: #294

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,38 +8,38 @@
...
 ### PipelineRun preview
-<div><img src="https://raw.githubusercontent.com/wiki/redhat-developer/vscode-tekton/images/readme/new/pipeline_run_preview_0.0.7.gif" width="710" height="499" /></div>
+<div><img src="https://raw.githubusercontent.com/wiki/redhat-developer/vscode-tekton/images/readme/new/pipeline_run_preview_0.0.7.gif" width="800" height="450" /></div>
 
 ### Pipeline preview
-<div><img src="https://raw.githubusercontent.com/wiki/redhat-developer/vscode-tekton/images/readme/new/pipeline_preview_0.0.6.gif" width="710" height="499" /></div>
+<div><img src="https://raw.githubusercontent.com/wiki/redhat-developer/vscode-tekton/images/readme/new/pipeline_preview_0.0.6.gif" width="600" height="493" /></div>
 
 ### New icons and Tekton resources
 <div><img src="https://raw.githubusercontent.com/wiki/redhat-developer/vscode-tekton/images/readme/new/new_items_and_icons.png" width="271" height="327" /></div>
```
Comment (sudhirverma) [inline README.md:39]: we should provide new screenshot for all resources

File: README.md [hunk 2]
```diff
@@ -8,38 +8,38 @@
 ## Overview
 
 A Visual Studio Code extension for interacting with Tekton Pipelines.
-<div><img src="https://raw.githubusercontent.com/wiki/redhat-developer/vscode-tekton/images/readme/demo_0.0.6.gif" width="710" height="499" /></div>
+<div><img src="https://raw.githubusercontent.com/wiki/redhat-developer/vscode-tekton/images/readme/demo_0.0.6.gif" width="800" height="500" /></div>
```
Comment (sudhirverma) [inline README.md:11]: you can update the new gif for interacting with pipelines.

File: README.md [hunk 3]
```diff
@@ -8,38 +8,38 @@
...
 ### Pipeline preview
-<div><img src="https://raw.githubusercontent.com/wiki/redhat-developer/vscode-tekton/images/readme/new/pipeline_preview_0.0.6.gif" width="710" height="499" /></div>
+<div><img src="https://raw.githubusercontent.com/wiki/redhat-developer/vscode-tekton/images/readme/new/pipeline_preview_0.0.6.gif" width="600" height="493" /></div>
 
 ### New icons and Tekton resources
 <div><img src="https://raw.githubusercontent.com/wiki/redhat-developer/vscode-tekton/images/readme/new/new_items_and_icons.png" width="271" height="327" /></div>
 
 ### Provides editing support for Pipeline yaml
-<div><img src="https://raw.githubusercontent.com/wiki/redhat-developer/vscode-tekton/images/readme/editing-demo.gif" width="710" height="499" /></div>
+<div><img src="https://raw.githubusercontent.com/wiki/redhat-developer/vscode-tekton/images/readme/editing-demo.gif" width="800" height="450" /></div>
```
Comment (sudhirverma) [inline README.md:42]: you can update new gif for editing support for pipeline yaml

**Comment by evidolob** [comment]
@sudhirverma In this PR I just want to fix current readme, not a provide new images/demos.
I will prepare new on release preparation.

**Comment by sudhirverma** [comment]
okay, I just provide that comment because the issue you tag there is `image present is distorted` description.

**Pull request merged**